include/labwc.h

```diff
@@ -527,6 +527,8 @@ void server_init(struct server *server);
 void server_start(struct server *server);
 void server_finish(struct server *server);
 
+void server_handle_view_destroy(struct server *server, struct view *view);
```

Comment:
Please rename to `view_handle_destroy` and move the implementation to `src/view.c`